README.md

```diff
@@ -182,6 +182,10 @@ Here's a list of supported alternative widgets for different JSONSchema data typ
   * `password`: an `input[type=password]` element;
   * by default, a regular `input[type=text]` element is used.
 
+The built-in string field also supports the JSONSchema `format` property, and will render an appropriate widget by default for the following formats:
+
+- `date-time`: An `input[type=datetime]` will be rendered.
```

Comment:
> Should we remove the dash : `datetime` ?

Well, `date-time` is what's expected by the JSONSchema spec, so nope.

> Other formats like email, password, color or url would be trivial to contribute

Thi is actually my next followup PR ;)